### PR TITLE
Update: AAT v1 readyness (fixes: #23)

### PR DESCRIPTION
**Pull request by joe-replin**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Update
* Fixes: #23 
* Adds v1 AAT schemas
* Bower Normalization
* ES Linting

[//]: # (List appropriate steps for testing if needed)
### Testing
1. Upload to AAT to ensure compatibility

[//]: # (Mention any other dependencies)
### Blockers
Currently unable to test in v1 AAT.

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Comment by joe-replin** [comment]
Tested in v1 AAT. Updated schemas & bower files.

**Review by guywillis — CHANGES_REQUESTED** [review]
ReadMe amends:

* L24 is on a new line when it doesn't need to be
* L27 convert double return to single
* L38 convert double return to single
* L40 `_setCompletionOnView` needs adding to the list
* L57 convert double return to single
* L66 / L67 / L68 add `<br>` at the end of each line

File: bower.json [hunk 1]
```diff
@@ -1,26 +1,23 @@
 {
   "name": "adapt-vimeo",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/adaptlearning/adapt-vimeo.git"
-  },
   "version": "3.1.1",
   "framework": ">=5.19.1",
   "homepage": "https://github.com/adaptlearning/adapt-vimeo",
   "issues": "https://github.com/adaptlearning/adapt-vimeo/issues/new",
```
Comment (guywillis) [inline bower.json:6]: Can `/new` be removed from this to mirror other plugins please

File: schema/component.schema.json [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            "play",
+            "ended"
+          ],
+          "_backboneForms": "Select"
+        },
+        "_media": {
+          "type": "object",
+          "title": "Media settings",
+          "properties": {
+            "_source": {
```
Comment (guywillis) [inline schema/component.schema.json:47]: Should the `_source` object have a placeholder like the [youtube](https://github.com/adaptlearning/adapt-youtube/blob/master/properties.schema#L60) version?

File: schema/course.schema.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          "type": "object",
+          "default": {},
+          "properties": {
+            "_vimeo": {
+              "type": "object",
+              "default": {},
+              "properties": {
+                "ariaRegion": {
+                  "type": "string",
+                  "title": "Aria region",
```
Comment (guywillis) [inline schema/course.schema.json:21]: I think we should prefix the title here `Vimeo:` so it's more specific in the course settings 

Screenshot 2023-02-23 at 16 26 59

File: schema/course.schema.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+                  "default": "This is a media component which displays a Vimeo video. Select the play / pause button to watch it."
+                },
+                "skipToTranscript": {
+                  "type": "string",
+                  "title": "Skip to transcript",
+                  "default": "Skip to transcript"
```
Comment (guywillis) [inline schema/course.schema.json:27]: Missing `translatable` property 

[example](https://github.com/adaptlearning/adapt-contrib-media/blob/master/schema/course.schema.json#L32)

**Comment by joe-replin** [comment]
> ReadMe amends:
> 
>     * L24 is on a new line when it doesn't need to be
> 
>     * L27 convert double return to single
> 
>     * L38 convert double return to single
> 
>     * L40 `_setCompletionOnView` needs adding to the list
> 
>     * L57 convert double return to single
> 
>     * L66 / L67 / L68 add `<br>` at the end of each line

Done! `_setCompletionOnView` is already added to the transcript section.

File: schema/component.schema.json [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+  "$merge": {
+    "source": {
+      "$ref": "component"
+    },
+    "with": {
+      "required": [
+        "_media"
+      ],
+      "properties": {
+        "_supportedLayout": {
```
[the remark this answers is not in the document]
Comment (joe-replin) [reply schema/component.schema.json:14]: Added.

File: schema/component.schema.json [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+        },
+        "_transcript": {
+          "type": "object",
+          "title": "Transcript Options",
+          "default": {},
+          "properties": {
+            "_setCompletionOnView": {
+              "type": "boolean",
+              "title": "Trigger completion?",
+              "description": "Whether to have this component mark as completed when the transcript is shown.",
```
[the remark this answers is not in the document]
Comment (joe-replin) [reply schema/component.schema.json:80]: I've synced all transcript options with that is present in the media component.

File: bower.json (hunk 1 above)
Comment (joe-replin) [reply bower.json:6]: Done

File: package.json [hunk 7]
```diff
@@ -1,26 +1,23 @@
 {
   "name": "adapt-vimeo",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/adaptlearning/adapt-vimeo.git"
-  },
   "version": "3.1.1",
   "framework": ">=5.19.1",
   "homepage": "https://github.com/adaptlearning/adapt-vimeo",
   "issues": "https://github.com/adaptlearning/adapt-vimeo/issues/new",
```
[the remark this answers is not in the document]
Comment (joe-replin) [reply package.json:6]: Done

File: schema/component.schema.json (hunk 2 above)
Comment (joe-replin) [reply schema/component.schema.json:47]: Good call.

File: schema/course.schema.json (hunk 3 above)
Comment (joe-replin) [reply schema/course.schema.json:21]: I'm on board.

File: schema/course.schema.json (hunk 4 above)
Comment (joe-replin) [reply schema/course.schema.json:27]: Good spot.

**Comment by joe-replin** [comment]
All of your recommendations have been included, @guywillis. Thanks for the thorough review.

**Review by guywillis — APPROVED** [review]
👍

**Pull request merged**